journal/modernized/08-chapter-8.adoc

```diff
@@ -399,15 +434,30 @@ be his portion, even now, in old age, agreeably to the words of the Psalmist:
 "`They that be planted in the house of the Lord, shall flourish in the courts of our God;
 they shall bring forth fruit in old age; they shall be fat and flourishing.`"
 
+[.signed-section-closing]
 I remain, your friend,
 
+[.signed-section-signature]
 Joseph Edgerton.
 
-Letter to T. B. G.:
+--
 
+[.asterism]
+'''
```

Comment:
I never mentioned this, but I try to be pretty sparing with asterisms. If I see two letters back to back, and second one has a `[.letter-heading]` then I feel like the letter heading is sufficient to break the two and make it clear the shift, so I tend to _not_ put an asterism there, or remove it if it was in the `.odt`. If you disagree we can definitely talk about it and go a different direction.